### PR TITLE
Half-Life 2 server heading tweaks, filename change

### PR DESCRIPTION
**Pull request by EdwardAngert**

### CODE REVIEW

File: docs/game-servers/install-a-half-life-2-deathmatch-dedicated-server-on-debian-or-ubuntu.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ This guide will show you how to set up your own [Half-Life 2 Deathmatch](http://
         sudo iptables -R INPUT 5 -p udp -m udp --sport 27000:27030 --dport 1025:65355 -j ACCEPT
         sudo iptables -I INPUT 7 -p udp -m udp --dport 27000:27030 -j ACCEPT
 
-## Install the Server
+## Install the Steam Server
```
Comment (davidebeatrici) [inline docs/game-servers/install-a-half-life-2-deathmatch-dedicated-server-on-debian-or-ubuntu.md]: **Server** should be removed, as SteamCMD is a Steam client.

File: docs/game-servers/install-a-half-life-2-deathmatch-dedicated-server-on-debian-or-ubuntu.md [hunk 2]
```diff
@@ -56,9 +56,9 @@ This guide will show you how to set up your own [Half-Life 2 Deathmatch](http://
 
         quit
 
-Occasionally, SteamCMD encounters an unreported error but completes successfully. If the steps in the Run the Server section are not successful, repeat the installation process to retrieve all of the game data, then run the server again.
+Occasionally, SteamCMD encounters an unreported error, but completes successfully. If the steps in the Run the Server section are not successful, repeat the installation process to retrieve all of the game data, then run the server again.
 
-## Run the Server
+## Run the Steam Server
```
Comment (davidebeatrici) [inline docs/game-servers/install-a-half-life-2-deathmatch-dedicated-server-on-debian-or-ubuntu.md]: Same as above.

File: docs/game-servers/install-a-half-life-2-deathmatch-dedicated-server-on-debian-or-ubuntu.md (hunk 1 above)
Comment (davidebeatrici) [inline docs/game-servers/install-a-half-life-2-deathmatch-dedicated-server-on-debian-or-ubuntu.md]: It's not a Steam Server.

File: docs/game-servers/install-a-half-life-2-deathmatch-dedicated-server-on-debian-or-ubuntu.md [hunk 3]
```diff
@@ -121,7 +121,7 @@ This script automatically starts your server in a Screen session.
 
         ./run.sh
 
-## Configure the server
+## Configure the Steam Server
```
Comment (davidebeatrici) [inline docs/game-servers/install-a-half-life-2-deathmatch-dedicated-server-on-debian-or-ubuntu.md]: Same as above.

File: docs/game-servers/install-a-half-life-2-deathmatch-dedicated-server-on-debian-or-ubuntu.md (hunk 1 above)
Comment (davidebeatrici) [reply docs/game-servers/install-a-half-life-2-deathmatch-dedicated-server-on-debian-or-ubuntu.md]: It's an Half-Life 2: Deathmatch server installed from Steam's command line client (SteamCMD).

Comment (kcraig34) [inline docs/game-servers/install-a-half-life-2-deathmatch-dedicated-server-on-debian-or-ubuntu.md]: Let's rephrase this H2 to "Install Server Using SteamCMD"

File: docs/game-servers/install-a-half-life-2-deathmatch-dedicated-server-on-debian-or-ubuntu.md (hunk 3 above)
Comment (kcraig34) [inline docs/game-servers/install-a-half-life-2-deathmatch-dedicated-server-on-debian-or-ubuntu.md]: Should this H2 now be "Configure the Half-Life 2 Server" if server is NOT accurately described as a Steam Server?

Comment (davidebeatrici) [reply docs/game-servers/install-a-half-life-2-deathmatch-dedicated-server-on-debian-or-ubuntu.md]: Yes, I think so.

File: docs/game-servers/install-a-half-life-2-deathmatch-dedicated-server-on-debian-or-ubuntu.md (hunk 1 above)
Comment (davidebeatrici) [reply docs/game-servers/install-a-half-life-2-deathmatch-dedicated-server-on-debian-or-ubuntu.md]: I agree.

**Comment by EdwardAngert** [comment]
thanks, @davidebeatrici and @kcraig34!

headings are fixed in the latest commit. If you're good with them, I'll merge

**Comment by kcraig34** [comment]
Merge away, @EdwardAngert !

**Pull request merged**